### PR TITLE
Update website info to AreaFiftyLAN 2017

### PR DESCRIPTION
**Pull request by MatthijsKok**
Location, prizes, tournaments and other info updated.

### CODE REVIEW

File: src/lancie-home-page/lancie-home-page.html [hunk 1]
```diff
@@ -86,19 +88,21 @@
...
           <br /><sub>*CH members/alumni will get a small discount</sub>
         </p>
+        <!--
         <p>
           Besides that you can play any game you like, there are organized activities during the weekend.
           Below you can find a global overview. Please note this is subject to change and of course this is not all!
           A final schedule will be provided during the event.
         </p>
         <lancie-timeline></lancie-timeline>
+        -->
```
Comment (martijnjanssen) [inline src/lancie-home-page/lancie-home-page.html]: You can just remove the entire timeline, we can update it and put it back in another time.

**Comment by MatthijsKok** [comment]
I agree these are incomplete, however we want to push this new information as fast as possible.

**Comment by TimvdLippe** [comment]
Then the other sections have to be updated too asap. We do not merge incomplete PRs

**Comment by martijnjanssen** [comment]
The entire homepage has to be updated, it is not needed to fill in all the information as of now. We can leave parts of them unfilled and not visible. But the parts that are staying should be updated and adapted for the coming event.

**Pull request closed without merging**